runtime/onert/api/python/package/experimental/train/optimizer/sgd.py

```diff
@@ -0,0 +1,21 @@
+from .optimizer import Optimizer
+
+
+class SGD(Optimizer):
+    """
+    Stochastic Gradient Descent (SGD) optimizer.
+    """
+    def __init__(self, learning_rate=0.001, momentum=0.0):
+        """
+        Initialize the SGD optimizer.
+        Args:
+            learning_rate (float): The learning rate for optimization.
+            momentum (float): Momentum factor (default: 0.0).
+        """
+        super().__init__(learning_rate)
+
+        if momentum != 0.0:
+            raise NotImplementedError(
+                "Momentum is not supported in the current version of SGD.")
+        self.momentum = momentum
+        self.velocity = None
```

Comment:
maybe...? this `velocity` variable looks like a legacy from the first implementation. 

```suggestion
```

Reply:
That might be used later, but it's not needed now.

Reply:
@zetwhite 
Do you prefer to remove it now?